setup.py

```diff
@@ -48,6 +48,12 @@
     zip_safe=False,
 
     install_requires=[
+        'collective.z3cform.widgets',
+        'ftw.simplelayout [contenttypes]',
+        'plone.api',
+        'plone.app.dexterity',
+        'plone.dexterity',
```

Comment:
Can you take a look at the ``bin/dependencychecker`` output and update the dependencies?
Be aware that the dependencychecker is not always right :wink: 
https://jenkins.4teamwork.ch/view/All/job/ftw.news-pull-request-1-test-plone-4.3.x.cfg/Dependency_Report/